<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDetailsTrk</class>
 <widget class="QWidget" name="IDetailsTrk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="CPlotProfile" name="plotElevation" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CPlotSpeed" name="plotSpeed" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CPlotDistance" name="plotDistance" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="comboColor"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineName"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkProfile">
           <property name="text">
            <string>Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkSpeed">
           <property name="text">
            <string>Speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkProgress">
           <property name="text">
            <string>Progress</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTainted">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../resources.qrc">:/icons/32x32/Tainted.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolLock">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources.qrc">
             <normaloff>:/icons/32x32/UnLock.png</normaloff>
             <normalon>:/icons/32x32/Lock.png</normalon>:/icons/32x32/UnLock.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string notr="true">#</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ele.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Delta</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Dist.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Speed</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ascend</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Descend</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Position</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelInfo">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelInfoTrkPt">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPlotProfile</class>
   <extends>QWidget</extends>
   <header>plot/CPlotProfile.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CPlotDistance</class>
   <extends>QWidget</extends>
   <header>plot/CPlotDistance.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CPlotSpeed</class>
   <extends>QWidget</extends>
   <header>plot/CPlotSpeed.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
